src/database/index.js

```diff
@@ -0,0 +1,18 @@
+import dotenv from "dotenv";
```

Comment:
Check your database folder structure and the naming conventions: create a config folder in the database folder, move the index.js file to the database>config folder, and rename the index.js file config.js.